### DISCUSSION TITLE
Let us disable /pt

### DISCUSSION THREAD

**Discussion by IveMalfunctioned** [GriefPrevention/GriefPrevention#1711] [Ideas]
/pt interferes with the essentials /pt command and it's quite annoying, please let us disable it

**Comment by Jikoo** [comment]
You can use your `commands.yml` to redirect commands to specific aliases. In this case you'd want to direct `/pt` to `/essentials:pt`. Alternately, I believe Essentials has its own settings to attempt to block plugins from overriding its commands.
